src/gymnasium_interface/example.py

```diff
@@ -0,0 +1,54 @@
+import logging
+from gymnasium_interface.pycram_gym_env import PyCRAMGymEnv
+from pycram.datastructures.enums import Arms, Grasp
+from pycram.datastructures.pose import Pose
+
+# Configure logging
+logging.basicConfig(level=logging.INFO, format="%(asctime)s - %(levelname)s - %(message)s")
+
+def custom_reward(state):
+    """
+    Custom reward function.
+
+    :param state: The current state of the environment.
+    :type state: dict
+    :return: Reward value based on the state.
+    :rtype: float
+    """
+    return 10.0 if state else -1.0
+
+# Define actions as a list of strings
+actions = ["navigate", "pick_up"]
+
+# Define default parameters for each action
+default_params = {
+    "navigate": {"target_pose": Pose(position=[1.0, 2.0, 0.0], orientation=[0.0, 0.0, 0.0, 1.0])},
+    "pick_up": {"object_desig": "milk", "arm": Arms.RIGHT, "grasps": [Grasp.FRONT]},
+}
+
+# Define objects to initialize in the environment
+objects = [
```

Comment:
this needs some more engineering. Perhaps objects that do not have worlds yet? Please prepare a short discussion of this kind of problem for the next pycram meeting